tests/llama_index/test_llama_index_tracer.py

```diff
@@ -443,3 +446,98 @@ def test_tracer_handle_tracking_uri_update(tmp_path):
         # The new trace will be logged to the updated tracking URI
         OpenAI().complete("Hello")
         assert len(_get_all_traces()) == 1
+
+
+@pytest.mark.skipif(
+    llama_core_version >= Version("0.11.10"),
+    reason="Workflow tracing does not work correctly in >= 0.11.10 until "
+    "https://github.com/run-llama/llama_index/issues/16283 is fixed",
+)
+@pytest.mark.skipif(
+    Version(llama_core_version) < Version("0.11.0"),
+    reason="Workflow was introduced in 0.11.0",
+)
+@pytest.mark.asyncio
+async def test_tracer_simple_workflow():
+    from llama_index.core.workflow import StartEvent, StopEvent, Workflow, step
+
+    class MyWorkflow(Workflow):
+        @step
+        async def my_step(self, ev: StartEvent) -> StopEvent:
+            # do something here
+            return StopEvent(result="Hi, world!")
+
+    w = MyWorkflow(timeout=10, verbose=False)
+    await w.run()
+
+    traces = _get_all_traces()
+    assert len(traces) == 1
+    assert traces[0].info.status == TraceStatus.OK
+    for s in traces[0].data.spans:
+        assert s.status.status_code == SpanStatusCode.OK
+    assert all(s.status.status_code == SpanStatusCode.OK for s in traces[0].data.spans)
```

Comment:
Can we remove the for loop and just keep the `all()` assertion?

Reply:
Oh I don't even know why it's still there, probably some rebase madness. Thanks for the catch!